### DISCUSSION TITLE
Hotspot: button position is wrong in the HotspotAnnotation

### DISCUSSION THREAD

**Discussion by llllkatjallll** [google/model-viewer#3037] [Q&A]
### Description

Situation: 
I have a hotspot and annotation-box as a child of this hotspot. In my annotation-box I want to place text, and a button. When I add a button to my annotation-box, the button appears not IN the annotation-box but in the top-left corner of the model-viewer box.
<img width="450" alt="chrome_f20wkEMLLR" src="https://user-images.githubusercontent.com/38395689/145050335-e433592b-ab4c-4de6-8905-6f3ba009c2f0.png">
<img width="694" alt="Code_tWUD4uIvqi" src="https://user-images.githubusercontent.com/38395689/145050643-85570a60-3ca2-4447-aa77-cae5aab64b1c.png">

If I do the same with other elements like div or li the position is right (in the annotation box under the text)
<img width="450" alt="chrome_OGduUqq5x4" src="https://user-images.githubusercontent.com/38395689/145050602-746d4b68-d3a5-4b50-9650-1da6d7b6782c.png">
<img width="742" alt="Code_5FsBYzfTRh" src="https://user-images.githubusercontent.com/38395689/145050689-e3302a80-8ef5-4e9d-9283-c16caad8784b.png">

#### Live Demo

Live Demo is here: https://prefrontalcortex.de/labs/model-viewer/button-in-annotation/

### Version
model-viewer : v1.9.2

### Tested in:

- [ x] Chrome, version: 96.0.4664.45
- [ ] Edge
- [ x] Firefox, 94.0.2 (64-Bit)
- [ ] IE
- [ ] Safari

**Comment by elalish** [comment]
That's bizarre, but I don't think it's a MV thing. All  we do is adjust the transform on  the parent element; the rest of the  HTML  rendering is  up  to the browser. Are you sure you don't have some kind of  CSS that's conflicting? Maybe an errant `display` property?
